infrastructure/quick-deploy/aws/variables.tf

```
@@ -490,6 +490,15 @@ variable "mongodb_efs" {
   default = null
 }
 
+variable "TF_VAR_mongodb_atlas" {
```

```suggestion
variable "mongodb_atlas" {
```

You don't need to prefix your variable with `TF_VAR_`: https://developer.hashicorp.com/terraform/cli/config/environment-variables#tf_var_name